dali/python/nvidia/dali/pipeline.py

```diff
@@ -116,13 +118,29 @@ class Pipeline(object):
 `enable_memory_stats`: bool, optional, default = False
     If DALI should print operator output buffer statistics.
     Usefull for `bytes_per_sample_hint` operator parameter.
+`py_num_workers`: int, optional, default = None
+    Specify the number of Python workers that will process ExternalSource callbacks.
+    If a number is not specified, the pool will use the cpu count as the number of Python workers.
+    The pool starts only if there is at least one ExternalSource with ``parallel`` set to True.
+    Setting it to 0 disables the pool and all ExternalSource operators fall back to non parallel
+    mode even if ``parallel`` is set to True.
+`py_start_method` : str, default = "fork"
+    Specify how Python workers should start. Supported methods:
```

Comment:
```suggestion
    Determines how Python workers are started. Supported methods:
```

Reply:
Done. I thought so, but it turns out, they need more spacing around.